core/src/main/java/org/apache/calcite/plan/volcano/VolcanoPlanner.java

```diff
@@ -917,13 +917,13 @@ RelNode changeTraitsUsingConverters(
   public void setImportance(RelNode rel, double importance) {
     assert rel != null;
     if (importance == 0d) {
-      relImportances.put(rel, importance);
+      nodesToSkip.add(rel);
     }
   }
 
   Double getImportance(RelNode rel) {
     assert rel != null;
-    return relImportances.get(rel);
+    return nodesToSkip.contains(rel) ? 0d : null;
   }
```

Comment:
You can remove this method.

Reply:
Removed. Thanks for the good suggestion.